lib/services/publish.test.js

```diff
@@ -7,11 +7,10 @@ const _ = require('lodash'),
   db = require('../services/db'),
   lib = require(`./${filename}`);
 
-require('sinon-as-promised');
+// require('sinon-as-promised');
```

Comment:
can be removed?

Reply:
Yep! Will also remove. So much deprecation. This is a reminder to myself to also check that I removed `winston`